apps/worker/src/publisher/publish-event.listener.ts

```diff
@@ -0,0 +1,19 @@
+import { QueueEventsListener, QueueEventsHost, OnQueueEvent } from '@nestjs/bullmq';
+
+@QueueEventsListener('publishQueue')
+export class PublishEventListener extends QueueEventsHost {
+  startTime: number;
+
+  constructor() {
+    super();
+    this.startTime = new Date().getTime();
+  }
+
+  @OnQueueEvent('drained')
+  onDrained({ jobId }: { jobId: string }) {
+    // do some stuff
+
+    const elapsed = new Date().getTime();
+    console.log((elapsed - this.startTime) / 1000);
+  }
+}
```

Comment:
not sure we need this but keeping it for now

Reply:
I think we can remove it.